pypeit/spectrographs/shane_kast.py

```diff
@@ -413,6 +414,42 @@ def configuration_keys(self):
         # Add grating tilt
         return cfg_keys+['dispangle']
 
+    def config_specific_par(self, scifile, inp_par=None):
+        """
+        Modify the PypeIt parameters to hard-wired values used for
+        specific instrument configurations.
+
+        .. todo::
+            Document the changes made!
+
+        Args:
+            scifile (str):
+                File to use when determining the configuration and how
+                to adjust the input parameters.
+            inp_par (:class:`pypeit.par.parset.ParSet`, optional):
+                Parameter set used for the full run of PypeIt.  If None,
+                use :func:`default_pypeit_par`.
+
+        Returns:
+            :class:`pypeit.par.parset.ParSet`: The PypeIt paramter set
+            adjusted for configuration specific parameter values.
+        """
+        par = self.default_pypeit_par() if inp_par is None else inp_par
+        # TODO: Should we allow the user to override these?
+
+        if self.get_meta_value(scifile, 'dispname') == '300/7500':
+            # TODO -- Note in docs that a NoNe solution is available too
+            par['calibrations']['wavelengths']['method'] = 'full_template'
+            par['calibrations']['wavelengths']['reid_arxiv'] = 'shane_kast_red_300_7500.fits'
+            # Add CdI
+            par['calibrations']['wavelengths']['lamps'] = ['NeI', 'HgI', 'HeI', 'ArI', 'CdI']
+        else:
+            pass
```

Comment:
Instead:
```
if self.get_meta_value(scifile, 'dispname') != '300/7500':
    return par

# TODO -- Note ...
...
return par
```

It's just a style issue, I guess, but I don't understand the `if something: else: pass` construct instead of just doing `if not something:`

Reply:
I was intending to do something (and still might someday)  ;)